.prettierignore

```diff
@@ -9,6 +9,7 @@ node_modules
 docs/chunks/
 docs/js/bundle.js
 docs/index.html
+README.md
```

Comment:
Not sure if we really want this, but the README was failing the prettier check for some reason. If there's a way around this please let me know.